keras_cv/layers/preprocessing/solarization.py

```diff
@@ -59,31 +64,36 @@ class Solarization(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
             or [height, width, channels].
     """
 
-    def __init__(self, value_range, addition=0.0, threshold=0.0, **kwargs):
-        super().__init__(**kwargs)
-        self.addition = addition
-        self.threshold = threshold
+    def __init__(self, value_range, addition_factor=0.0, threshold_factor=0.0, seed=None, **kwargs):
+        super().__init__(seed=seed, **kwargs)
+        self.addition_factor = preprocessing.parse_factor(
+            addition_factor, max_value=256, seed=seed, param_name="addition_factor"
```

Comment:
shouldn't the max_value be 255?

Reply:
I was thinking because we use the `<` operator so we should actually support all the way to 256.

Instead, let's opt to `<=` so we can keep the range to 255 as you suggested.  That is better, done.